src/components/views/SerialView/SerialView.js

```diff
@@ -202,6 +202,7 @@ const SerialView = ({
         ruleset={serial?.serialRulesets?.find(
           (sr) => sr?.rulesetStatus?.value === 'active'
         )}
+        serialName={serial?.orderLine?.title ?? serial?.description}
```

Comment:
Here the fallback should be `serial?.id` as opposed to `serial?.description` so its in line with what we do within the serials SASQRoute component

Reply:
I'm not sure @Jack-Golding 
In the JIRA it says:
`<serial title or description>` and not `<serial title or id>`
Or is the id the same as the description?

Reply:
@CalamityC @Jack-Golding my error on writing the story - I'd forgotten we fall back the UUID. I've updated the story @CalamityC and agree with Jack that this is the right thing to do